docs/reference/mapping/params/dynamic.asciidoc

```diff
@@ -1,24 +1,32 @@
 [[dynamic]]
 === `dynamic`
 
-By default, fields can be added _dynamically_ to a document, or to
-<<object,inner objects>> within a document, just by indexing a document
-containing the new field.  For instance:
+When you index a document containing a new field, {es} adds the
```

Comment:
Does this link to the page in the doc above? It feels like that'd be important.

Reply:
There's a link later in the page, but I'll move it here 👍